frontend/src/common/constants/httpStatusCode.ts

```diff
@@ -0,0 +1,381 @@
+// https://gist.github.com/scokmen/f813c904ef79022e84ab2409574d1b45
```

Comment:
Since enum is constant, let's use UPPER_CASE for both name and file!

e.g.,

`frontend/src/common/constants/HTTP_STATUS_CODE.ts`

```ts
enum HTTP_STATUS_CODE { ... }
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#basic_const_usage

Reply:
ah last thing is updating the file name! `frontend/src/common/constants/HTTP_STATUS_CODE.ts` thank you!